admin/installer/setup_client.sh

```diff
@@ -55,5 +55,12 @@ ${DOCKER_CERT_HOME}/createclient.exp
 # Copy Docker Certificates directory to Ubuntu's home directory.
 cp -r ${DOCKER_CERT_HOME} ${UBUNTU_HOME}
 
+# Clear out S3 bucket used by CloudFormation so that subsequent stack destroy attempt succeeds.
+s3cmd_wrapper del --recursive --config=/root/.s3cfg s3://${s3_bucket}
```

Comment:
I tried running this command manually and got:

```
root@ip-172-31-8-197:~# /usr/bin/s3cmd del --recursive --config=/root/.s3cfg s3://testinstallerstack1455564574-clusterconfig-vg6qjdd63zpi
ERROR: Parameter problem: Please use --force to delete ALL contents of s3://testinstallerstack1455564574-clusterconfig-vg6qjdd63zpi
root@ip-172-31-8-197:~# /usr/bin/s3cmd del --force --recursive --config=/root/.s3cfg s3://testinstallerstack1455564574-clusterconfig-vg6qjdd63zpi
File s3://testinstallerstack1455564574-clusterconfig-vg6qjdd63zpi/docker-swarm-tls-config/ca-key.pem deleted
File s3://testinstallerstack1455564574-clusterconfig-vg6qjdd63zpi/docker-swarm-tls-config/ca.pem deleted
File s3://testinstallerstack1455564574-clusterconfig-vg6qjdd63zpi/docker-swarm-tls-config/passphrase.txt deleted
File s3://testinstallerstack1455564574-clusterconfig-vg6qjdd63zpi/swarm-config/swarm_cluster_id deleted
```

I think the command needs to be:

```
/usr/bin/s3cmd del --force --recursive --config=/root/.s3cfg s3://${s3_bucket}
```